### PR TITLE
[android_alarm_manager] support for isAlarmActive

### PR DESCRIPTION
**Pull request by GadepalliAditya1998**
This pull request aims at adding a function / method that returns the status of the alarm whether it is active or not by returning a boolean. Use the function isAlarmActive(int requestCode).

### CODE REVIEW

File: packages/android_alarm_manager/example/android/build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.0'
+        classpath 'com.android.tools.build:gradle:3.4.2'
```
[the remark this answers is not in the document]
Comment (GadepalliAditya1998) [reply packages/android_alarm_manager/example/android/build.gradle]: When I Opened in Android studio it has changed the gradle files while syncing.

File: packages/android_alarm_manager/android/src/main/java/io/flutter/plugins/androidalarmmanager/AndroidAlarmManagerPlugin.java [hunk 2]
```diff
@@ -137,6 +137,12 @@ public void onMethodCall(MethodCall call, Result result) {
         int requestCode = ((JSONArray) arguments).getInt(0);
         AlarmService.cancel(mContext, requestCode);
         result.success(true);
+      } else if (method.equals("Alarm.isActive")) {
+        //This method indicates whether the alarm has already been set or not
```
[the remark this answers is not in the document]
Comment (GadepalliAditya1998) [reply packages/android_alarm_manager/android/src/main/java/io/flutter/plugins/androidalarmmanager/AndroidAlarmManagerPlugin.java]: I used flutter_plugin_tools tools to format it. Should I need to do it manually?

File: packages/android_alarm_manager/example/android/gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,6 +1,6 @@
-#Fri Jun 23 08:50:38 CEST 2017
+#Tue Aug 27 14:58:49 IST 2019
```
[the remark this answers is not in the document]
Comment (GadepalliAditya1998) [reply packages/android_alarm_manager/example/android/gradle/wrapper/gradle-wrapper.properties]: @bkonyi  Okay, actually this file was also added when i started editing in Android studio.

File: packages/android_alarm_manager/example/lib/main.dart [hunk 4]
```diff
@@ -5,27 +5,103 @@
...
+
+    AndroidAlarmManager.isAlarmActive(oneShotID).then((bool value) {
+      setState(() {
+        isOneShotActive = value;
+      });
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
```
[the remark this answers is not in the document]
Comment (GadepalliAditya1998) [reply packages/android_alarm_manager/example/lib/main.dart]: Capture


finally this was the issue.

**Comment by GadepalliAditya1998** [comment]
@bkonyi I am facing with this issue again.

image

File: packages/android_alarm_manager/android/src/main/java/io/flutter/plugins/androidalarmmanager/AndroidAlarmManagerPlugin.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (bkonyi) [reply packages/android_alarm_manager/android/src/main/java/io/flutter/plugins/androidalarmmanager/AndroidAlarmManagerPlugin.java]: I don't believe the Java formatter works on comments for whatever reason, so yes you'll have to add the space manually.

File: packages/android_alarm_manager/example/android/build.gradle (hunk 1 above)
[the remark this answers is not in the document]
Comment (bkonyi) [reply packages/android_alarm_manager/example/android/build.gradle]: Yeah, I'd revert the changes to this file then.

File: packages/android_alarm_manager/example/android/gradle/wrapper/gradle-wrapper.properties (hunk 3 above)
[the remark this answers is not in the document]
Comment (bkonyi) [reply packages/android_alarm_manager/example/android/gradle/wrapper/gradle-wrapper.properties]: I'd also revert the changes to this file.

**Comment by bkonyi** [comment]
@GadepalliAditya1998 I've re-run the failing checks to see if they'll go green. If they keep failing, you'll need to rebase your PR from master.

**Comment by bkonyi** [comment]
Closing due to inactivity. Feel free to reopen if you want to move forward with this PR.

**Pull request closed without merging**